docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+---
+title: "Queries for Horizon/RPC-like Data"
+sidebar_position: 50
+---
+
+[Horizon](/docs/data/horizon) and [RPC](docs/data/rpc) both provide API endpoints to retrieve data from the Stellar network. The following example queries try to retrieve the same data by using Hubble with the added benefit of being able to return historical data.
+
+## Accounts
+
+[horizon/accounts](/docs/data/horizon/api-reference/resources/accounts)
+
+Users interact with the Stellar network through accounts. Everything else in the ledger—assets, offers, trustlines, etc.—are owned by accounts, and accounts must authorize all changes to the ledger through signed transactions.
+
+Learn more about [accounts](../../../learn/glossary.mdx#account).
+
+### General account information and XLM balance
+
+```sql
+select
+  account_id
+  , sequence_number as sequence
+  , sequence_ledger
+  , sequence_time
+  , num_subentries
+  , home_domain
+  , last_modified_ledger
+  , balance
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.accounts`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### Non-XLM asset balance information
+
+```sql
+select
+  account_id
+  , asset_code
+  , asset_issuer
+  , asset_type
+  , balance
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.trust_lines`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### Account signer information
+
+```sql
+select
+  account_id
+  , signer
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.account_signers`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### List all accounts
+
+```sql
+select
+  account_id
+from `crypto-stellar.crypto_stellar_dbt.accounts_current`
+where true
+-- limit does not reduce the amount of data BigQuery queries
+limit 1000
+```
+
+### Retrieve an Account’s Transactions
+
+```sql
+select
+  op_account_id
+  , transaction_id
+  , transaction_hash
+  , ledger_sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
```

Comment:
It may be dangerous to encourage users to fetch transactions from the enriched table. You **must** group by to deduplicate transaction info with multiple operations. Either spell that out or switch the source

Reply:
Updated query